src/openforms/forms/api/serializers/form.py

```diff
@@ -168,6 +172,14 @@ class FormSerializer(PublicFieldsSerializerMixin, serializers.ModelSerializer):
     )
     hide_non_applicable_steps = serializers.SerializerMethodField(read_only=True)
     submission_report_download_link_title = serializers.SerializerMethodField()
+    submission_statements_configuration = serializers.SerializerMethodField(
```

Comment:
This name confused me a bit. I wonder if `formio_checkbox_statements`, or `config_checkbox_statements` would be clearer :thinking:

Reply:
I'm not 100% sure - we already have `submissionAllowed` which controls some parts of the summary page related to completing/submitting the form, so I wanted to have that `submission` part in there.

I'm also not sure about mentioning `formio` in there, as it's an implementation detail?

Maybe `submission_completion_checkboxes` would work, but if we ever get a different component in there than a checkbox, then that's also no longer an accurate statement. I'm kind of anticipating clients wanting to put whatever fields in there, e.g. they could request a signature component at some point too. Maybe `submission_completion_configuration` then? Which aligns with the `formStep.configuration` field which holds a formio definition.

Reply:
Ahh fair, it was the 'submission' part that confused me :smile: Then it can remain as it is!